prov/util/src/util_mr_cache.c

```diff
@@ -318,6 +319,39 @@ int ofi_mr_cache_search(struct ofi_mr_cache *cache, const struct fi_mr_attr *att
 	return ret;
 }
 
+struct ofi_mr_entry *ofi_mr_cache_peek(struct ofi_mr_cache *cache,
+				       const struct fi_mr_attr *attr)
```

Comment:
The current API is based on the names in search.c - tdelete, tsearch, tfind, etc.  To match, this would be cache_find.

Reply:
Alright, i'll change it to find